eduiddashboard/scripts/fallback_letter_proofing.py

```diff
@@ -0,0 +1,115 @@
+# -*- coding: utf-8 -*-
+
+import optparse
+import sys
+import textwrap
+from pyramid.paster import bootstrap
+from datetime import datetime
+import json
+
+from eduid_userdb.exceptions import UserOutOfSync
+from eduid_userdb.dashboard import DashboardLegacyUser as OldUser
+from eduiddashboard.verifications import get_verification_code, verify_nin
+from eduiddashboard.idproofinglog import LetterProofing
+from eduiddashboard.session import _get_user_by_eppn
+
+import logging
+logger = logging.getLogger(__name__)
+
+__author__ = 'lundberg'
+
+"""
+Verify a NIN for a letter verification that has failed due to a worker timeout.
+"""
```

Comment:
this script could use a wee bit more documentation here I think. what is the JSON data that is passed in as second argument for example?